init_test.go

```diff
@@ -26,10 +25,9 @@ func TestInitialization(t *testing.T) {
 
 		t.Parallel()
 
-		dir, err := ioutil.TempDir("", "badger-test")
+		file, err := ioutil.TempFile("", "sqlite-test")
```

Comment:
use `os.CreateTemp` instead

Reply:
fixed in [ca045c3](https://github.com/onflow/flow-emulator/pull/301/commits/ca045c39e8e2f727969e046a93c7ecde6d7ac3bb)